seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/sink/KafkaSinkWriter.java

```diff
@@ -129,6 +143,9 @@ private Properties getKafkaProperties(Config pluginConfig) {
         kafkaConfig.entrySet().forEach(entry -> {
             kafkaProperties.put(entry.getKey(), entry.getValue().unwrapped());
         });
+        if (pluginConfig.hasPath(ASSIGN_PARTITIONS)) {
+            kafkaProperties.put(ProducerConfig.PARTITIONER_CLASS_CONFIG, "org.apache.seatunnel.connectors.seatunnel.kafka.sink.CustomPartitioner");
+        }
```

Comment:
remove this code block



example:

```hocon
sink {
  kafka {
     ...
      kafka.partitioner.class = org.apache.seatunnel.connectors.seatunnel.kafka.sink.CustomPartitioner
  }  
}

you can add this example to docs
```

Reply:
Do you mean to let users write code to implement custom partitions?This may not simplify the operation of the user.
Because other operations are implemented through config configuration, code needs to be written here.

Reply:
This is just a more general way of configuring

Reply:
OK, I'll modify it in this way.